src/Sylius/Bundle/CoreBundle/Routing/RouteProvider.php

```diff
@@ -222,4 +223,26 @@ private function createRouteFromEntity($entity, $value = null)
 
         return new Route($this->routeConfigs[$className]['prefix'].'/'.$value, $defaults);
     }
+
+    /**
+     * @param string $identifier
+     * @param RepositoryInterface $repository
+     * @param string $className
+     *
+     * @return Object|null
+     */
+    private function tryToFindEntity($identifier, RepositoryInterface $repository, $className)
```

Comment:
These are not magic methods anymore. Also, this is just a temporary solution, I want to get rid of this routing anyway so this is just a temporary solution to keep the existing functionality working.